recipes/latentstrainanalysis/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "LatentStrainAnalysis" %}
+{% set version = "1" %}
```

Comment:
Is this an arbiratary version number or stated in the code? Maybe better to set it to `0.0.1` in case they will make a release below 1 later.